### PR TITLE
Alignment to fastq fix and add tests

### PR DESCRIPTION
**Pull request by ggabernet**
- Fixing a bug in `alignment_to_fastq.nf` subworkflow
- Adding tests for this subworkflow

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: conf/test.config [hunk 1]
```diff
@@ -48,37 +48,37 @@ params {
 
 profiles {
     annotation {
-        params.input               = "${baseDir}/tests/csv/3.0/vcf_single.csv"
+        params.input               = "${projectDir}/tests/csv/3.0/vcf_single.csv"
```
Comment (FriederikeHanssen) [inline conf/test.config:51]: I am fairly confused this hasn't caused any problems for us

Comment (maxulysse) [reply conf/test.config:51]: Same here

Comment (ggabernet) [reply conf/test.config:51]: yes, me too...

**Review by FriederikeHanssen — APPROVED** [review]
❤️

**Pull request merged**